jsonrest.go

```diff
@@ -307,7 +320,9 @@ func sendJSON(w http.ResponseWriter, status int, v interface{}) {
 	}
 
 	enc := json.NewEncoder(w)
-	enc.SetIndent("", "  ")
+	if r.jsonPrettyFormat {
+		enc.SetIndent("", "  ")
+	}
```

Comment:
And in that case this would become
```suggestion
	if !r.jsonPrettyFormatDisabled {
		enc.SetIndent("", "  ")
	}
```